engine/src/mbliphoneapp.mm

```diff
@@ -529,8 +529,22 @@ - (void)application:(UIApplication *)p_application didReceiveRemoteNotification:
 - (void)application:(UIApplication*)p_application didRegisterForRemoteNotificationsWithDeviceToken:(NSData*)p_device_token
 {
     NSString *t_to_log = [NSString stringWithFormat:@"%s%@%s", "Application: push notification device token (", p_device_token, ")"];
-    NSString *t_registration_text = [NSString stringWithFormat:@"%@", p_device_token];
+    NSString *t_registration_text;
     
+    NSUInteger t_length = p_device_token.length;
+    if (t_length == 0)
```

Comment:
Does this replicate exactly the old format? (Otherwise it will break people's code in the same way as the revised format from debugDescription does).

Reply:
hmm, not really. The old format was an alphanumeric string of length 64, bracketed by `< >`, displayed as 8 segments of 8 digits/chars each, separated by spaces.

The new format is just an alphanumeric string of length 64. Since the new format is already in the form people want to use, do we really want to keep the old format? Maybe we could add a release note saying that the token is already in the "final" form, and no further modifications are needed in the users' scripts?

Reply:
Example:
Old format: `<965b251c 6cb1926d e3cb366f dfb16ddd e6b9086a 8a3cac9e 5f857679 376eab7C>`

New format: `965b251c6cb1926de3cb366fdfb16ddde6b9086a8a3cac9e5f857679376eab7C`

Reply:
Okay - so we need to replicate the exact same format - otherwise it will break people's code (for much the same reasons as the change already did!)

Reply:
(We don't know what people do with the string they get)

Reply:
Ok sure 👍